django_spanner/base.py

```diff
@@ -123,6 +124,15 @@ def instance(self):
             project=os.environ["GOOGLE_CLOUD_PROJECT"]
         ).instance(self.settings_dict["INSTANCE"])
 
+    @property
+    def allow_transactions_in_auto_commit(self):
+        if "ALLOW_TRANSACTIONS_IN_AUTO_COMMIT" in self.settings_dict:
+            return self.settings_dict["ALLOW_TRANSACTIONS_IN_AUTO_COMMIT"]
+        if USING_DJANGO_3:
+            return False
+        if USING_DJANGO_4:
```

Comment:
nit: remove this if statement and just return true. That makes this work by default if we in the future support Django 5 or higher.

Reply:
Done